src/components/VHeader/VSearchBar/VSearchBar.vue

```diff
@@ -32,6 +32,7 @@ import { useMatchHomeRoute } from '~/composables/use-match-routes'
 import VInputField from '~/components/VInputField/VInputField.vue'
 import VSearchButton from '~/components/VHeader/VSearchBar/VSearchButton.vue'
 
+const SIZES = ['small', 'medium', 'large', 'standalone']
```

Comment:
Given these are passed straight through to the VInputField, could we import the array of sizes from there instead?

Reply:
Thank you for this suggestion! I have forgot that we are re-using the same array. Edited in the last commit.